src/hooks/validation/MultiSignerValidatorHook.sol

```diff
@@ -31,7 +31,7 @@ contract MultiSignerValidatorHook is IHook {
         bytes32 signerKeyHash = signerKey.hash();
 
         keyStorage[signerKeyHash] = encodedKey;
-        requiredSigners[_accountKeyHash(keyHash)].add(signerKeyHash);
+        requiredSigners[keyHash.wrap()].add(signerKeyHash);
```

Comment:
this is interesting, so the added signer HAS to be an secpk1 key in this impl

Reply:
yeah just for brevity for now